commutative-semigroups.cabal

```diff
@@ -1,10 +1,9 @@
-cabal-version:       2.4
 name:                commutative-semigroups
 version:             0.2
 synopsis:            Commutative semigroups
 description:
   A commutative semigroup is a semigroup where the order of arguments to mappend does not matter.
-license:             BSD-3-Clause
+license:             BSD3
```

Comment:
This was no typo. See https://spdx.org/licenses/

Reply:
I see. I saw build errors complaining about unknown license name, but today I wasn't able to reproduce error.